pkgs/command/flags_test.go

```diff
@@ -0,0 +1,21 @@
+package command
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestApplyFlagToFieldReflectString(t *testing.T) {
```

Comment:
You can add a `t.Parallel()` call as the first line of this test to make it run in parallel with other tests in the codebase

Reply:
Resolved-by: [b6750ae](https://github.com/gnolang/gno/pull/488/commits/b6750aef260e4ed307844436ddfdfaf02c99db2b)